modules/Book_Management.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable  no-restricted-globals */
```

Comment:
- [ ] please avoid turning off the linters in your project, linters are used to optimize your code and make it more readable, therefore we advise you to keep them turned on.